src/Workspaces/VisualBasic/Portable/Rename/VisualBasicRenameRewriterLanguageService.vb

```diff
@@ -275,6 +277,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Rename
                             suffix = "Event"
                         End If
 
+                        If fieldSymbol.AssociatedSymbol IsNot Nothing AndAlso
+                           fieldSymbol.AssociatedSymbol.IsKind(SymbolKind.Property) AndAlso
+                           fieldSymbol.Name = "_" + fieldSymbol.AssociatedSymbol.Name Then
```

Comment:
Is this check necessary? Is there places where the compiler may give fieldSymbols w/ associated symbols that aren't right?